pkg/kubernetes/informers.go

```diff
@@ -90,9 +103,14 @@ func (im *InformerManager) AddPVListener(add func(obj interface{}), update func(
 }
 
 // AddConfigMapListener hooks up add, update, delete callbacks
-func (im *InformerManager) AddConfigMapListener(add func(obj interface{}), update func(oldObj, newObj interface{}), remove func(obj interface{})) {
+func (im *InformerManager) AddConfigMapListener(ctx context.Context, client clientset.Interface, name string, namespace string, add func(obj interface{}), update func(oldObj, newObj interface{}), remove func(obj interface{})) {
 	if im.configMapInformer == nil {
-		im.configMapInformer = im.informerFactory.Core().V1().ConfigMaps().Informer()
+		// tweakListOptions helps in defining specific List Options for the ListFunc and WatchFunc handlers for the configmap informer
+		tweakListOptions := func(options *metav1.ListOptions) {
+			cmFieldSelector := fields.ParseSelectorOrDie(fmt.Sprintf("metadata.name=%s", name))
```

Comment:
Can't we simply do `options.FieldSelector = fmt.Sprintf("metadata.name=%s", name)`? Why do we need `ParseSelectorOrDie` here?

Reply:
Makes sense. `ParseSelectorOrDie` will panic during error, that's what I read from the docs.
However based on another review comment below, we don't need this at all, since we need a watch only on namespace and not both name & namespace. Because TKG has 2 FSS cMaps